metabase_api/metabase_api.py

```diff
@@ -383,7 +383,8 @@ def friendly_names_is_disabled(self):
             return True
 
         friendly_name_setting = [ i['value'] for i in self.get('/api/setting') if i['key'] == 'humanization-strategy' ][0]
-        return friendly_name_setting == 'none'  # 'none' means disabled
+        return friendly_name_setting == 'none' or friendly_name_setting is None  # 'none' means disabled
```

Comment:
Is there a scenario that `friendly_name_setting is None` becomes true?
I don't see the reason for adding that part.